autogpt/chat.py

```diff
@@ -52,7 +52,13 @@ def generate_context(prompt, relevant_memory, full_message_history, model):
 
 # TODO: Change debug from hardcode to argument
```

Comment:
Is this TODO comment is needs to stay if this gets merged?

Reply:
yea, the debug mode is still hardcoded